From 59b6f52c5290521ce2bd6209759e0d055c4bdc09 Mon Sep 17 00:00:00 2001
From: Myy <myy@miouyouyou.fr>
Date: Tue, 18 Jul 2017 02:21:44 +0000
Subject: [PATCH] Define VPU services in the Rockchip 3288 DTS files

And use them in the MiQi, Tinkerboard and Firefly board.

These will be used by the external Vcodec driver.

Signed-off-by: Myy <myy@miouyouyou.fr>
---
 arch/arm/boot/dts/rk3288-firefly.dtsi |  8 +++
 arch/arm/boot/dts/rk3288-miqi.dts     |  8 +++
 arch/arm/boot/dts/rk3288-tinker.dts   |  9 ++++
 arch/arm/boot/dts/rk3288.dtsi         | 97 +++++++++++++++++++++++++++++++++++
 4 files changed, 122 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-firefly.dtsi b/arch/arm/boot/dts/rk3288-firefly.dtsi
index 32dabae..13c8b42 100644
--- a/arch/arm/boot/dts/rk3288-firefly.dtsi
+++ b/arch/arm/boot/dts/rk3288-firefly.dtsi
@@ -612,3 +612,11 @@
 &wdt {
 	status = "okay";
 };
+
+&hevc_service {
+	status = "okay";
+};
+
+&vpu_service {
+	status = "okay";
+};
diff --git a/arch/arm/boot/dts/rk3288-miqi.dts b/arch/arm/boot/dts/rk3288-miqi.dts
index 30e93f6..c165eec 100644
--- a/arch/arm/boot/dts/rk3288-miqi.dts
+++ b/arch/arm/boot/dts/rk3288-miqi.dts
@@ -462,3 +462,11 @@
 &wdt {
 	status = "okay";
 };
+
+&hevc_service {
+	status = "okay";
+};
+
+&vpu_service {
+	status = "okay";
+};
diff --git a/arch/arm/boot/dts/rk3288-tinker.dts b/arch/arm/boot/dts/rk3288-tinker.dts
index f601c78..6bbc8a3 100644
--- a/arch/arm/boot/dts/rk3288-tinker.dts
+++ b/arch/arm/boot/dts/rk3288-tinker.dts
@@ -534,3 +534,12 @@
 &wdt {
 	status = "okay";
 };
+
+&hevc_service {
+	status = "okay";
+};
+
+&vpu_service {
+	status = "okay";
+};
+
-- 
2.10.2

